tests/lightgbm/test_LightGbmTreeEnsembleConverters.py

```diff
@@ -33,6 +35,33 @@ def test_lightgbm_regressor2(self):
         model = LGBMRegressor(n_estimators=2, max_depth=1, min_child_samples=1)
         dump_single_regression(model, suffix="2")
 
+    def test_lightgbm_booster_classifier(self):
+        X = [[0, 1], [1, 1], [2, 0], [1, 2]]
+        X = numpy.array(X, dtype=numpy.float32)
+        y = [0, 1, 0, 1]
+        data = lightgbm.Dataset(X, label=y)
+        model = lightgbm.train({'boosting_type': 'gbdt', 'objective': 'binary',
+                                'n_estimators': 3, 'min_child_samples': 1},
+                               data)
+        model_onnx, prefix = convert_model(model, 'tree-based multi-output classifier',
+                                           [('input', FloatTensorType([1, 2]))])
+        dump_data_and_model(X, model, model_onnx,
+                            allow_failure="StrictVersion(onnx.__version__) < StrictVersion('1.3.0')",
+                            basename=prefix + "Bin" + model.__class__.__name__)
+
+    def test_lightgbm_booster_regressor(self):
+        X = [[0, 1], [1, 1], [2, 0]]
+        X = numpy.array(X, dtype=numpy.float32)
+        y = [0, 1, 1.1]
+        data = lightgbm.Dataset(X, label=y)
+        model = lightgbm.train({'boosting_type': 'gbdt', 'objective': 'regression',
+                                'n_estimators': 3, 'min_child_samples': 1, 'max_depth': 1},
+                               data)
+        model_onnx, prefix = convert_model(model, 'tree-based binary classifier',
+                                           [('input', FloatTensorType([1, 2]))])
+        dump_data_and_model(X, model, model_onnx,
+                            basename=prefix + "Bin" + model.__class__.__name__)
```

Comment:
You can write basename=prefix + "BoosterBin" + model.__class__.__name__. The name is not really important except when the test fails. This name is used to dump everything on disk and investigate later.

Reply:
Renamed